### PR TITLE
ref(rust): Remove Python processors in favor of Rust ones

### PR DESCRIPTION
**Pull request by phacops**
With a fixed spans config.

### CODE REVIEW

File: snuba/datasets/configuration/spans/entities/spans.yaml [hunk 1]
```diff
@@ -59,28 +59,25 @@ schema:
         },
     },
     {
-      name: sentry_tags,
```
Comment (volokluev) [inline snuba/datasets/configuration/spans/entities/spans.yaml:62]: I still see plenty of config changes

**Comment by phacops** [comment]
I think the fix to the issue you're having is here https://github.com/getsentry/sentry/pull/62852

**Review by untitaker — APPROVED** [review]
let's merge https://github.com/getsentry/sentry/pull/62852 first and ensure that the previously failed sentry tests pass (locally)

File: tests/consumers/test_schemas.py [hunk 2]
```diff
@@ -42,6 +42,8 @@ def _generate_tests() -> Iterator[Case]:
 
         try:
             for example in sentry_kafka_schemas.iter_examples(topic.value):
+                if "replay" in str(example.path):
+                    continue
```
Comment (lynnagara) [inline tests/consumers/test_schemas.py:46]: @cmanallen this also needs to be reverted once replays is fixed

Comment (cmanallen) [reply tests/consumers/test_schemas.py:46]: Its reverted in this PR https://github.com/getsentry/snuba/pull/5342

Comment (cmanallen) [reply tests/consumers/test_schemas.py:46]: Oh wait no its not... I swear I removed that line of code.  Must have been a change to the master branch.  I will remove it when the time comes 👍

**Comment by phacops** [comment]
> let's merge [getsentry/sentry#62852](https://github.com/getsentry/sentry/pull/62852) first and ensure that the previously failed sentry tests pass (locally)

I tested locally and Sentry tests for DDM pass against this PR.

**Review by lynnagara — APPROVED** [review]

**Pull request merged**